gremlin/ogma-neptune/README.md

```diff
@@ -0,0 +1,248 @@
+
+## PREREQUISITES:
+
+1. Amazon Neptune Cluster
+2. Access to AWS IAM Permissions for
+   -> creating AWS Lambda functions
+   -> creating IAM roles for Amazon Neptune cluster to access S3 and, for API Gateway to access AWS Lambda functions
+   -> APIs in Amazon API Gateway
+   -> Amazon S3 buckets
+   -> creating and attaching VPC Endpoints to the VPC
+3. Ogma library
+
+## Architecture
+
+Below is the architecture that we will be using to visualize data in Amazon Neptune using [Ogma](https://ogma.linkurio.us).
+Since Amazon Neptune instance can't be accessed outside the VPC, we are using AWS Lambda function in VPC. This AWS Lambda function is accessed through the proxy created and exposed to internet using Amazon API Gateway.
+Once the proxy is exposed, we can access the APIs from Javascript code being executed from Amazon S3 (static website).
+
+![](images/architecture.png)
+
+## Steps
+
+1. [Provision Amazon Neptune Cluster](#1-provision-amazon-neptune-cluster)
+2. [Load sample data into Amazon Neptune](#2-load-sample-data-into-amazon-neptune)
+3. [Create and Configure AWS Lambda Function](#3-create-and-configure-aws-lambda-function)
+4. [Create and Configure Amazon API Gateway - Proxy API](#4-create-and-configure-amazon-api-gateway---proxy-api)
+5. [Configure Amazon S3 bucket for hosting a static website](#5-configure-amazon-s3-bucket-for-hosting-a-static-website)
+
+### 1. Provision Amazon Neptune Cluster
+
+Follow the procedure mentioned in [Amazon Neptune Documentation](https://docs.aws.amazon.com/neptune/latest/userguide/get-started-prerequisites.html) to provision a new Amazon Neptune Instance inside a VPC.<BR/>
+
+### 2. Load sample data into Amazon Neptune
+
+This GitHub repo provides you [sample twitter data](sampledata/) that you can load into Amazon Neptune.
+To load the data into Amazon Neptune, you will need to <br>
+
+- copy the sample data to Amazon S3
+- create IAM Role for accessing S3 bucket and attach it to Amazon Neptune cluster
+- create Amazon S3 VPC Endpoint and attach it to the VPC
+- create a "bastion host" inside the same VPC to load data from Amazon S3 into Amazon Neptune.<br/>
+
+For details on how to load data from Amazon S3 to Amazon Neptune please refer [this](https://docs.aws.amazon.com/neptune/latest/userguide/bulk-load.html) link from AWS documentation.
+
+### 3. Create and configure AWS Lambda function
+
+Once you load data into Amazon Neptune, you need to create the AWS Lambda function to access this data and expose it over RESTful interface through Amazon API Gateway.<br/>
+Execute below steps from the terminal to create the deployment package and create AWS Lambda function -
+
+```
```

Comment:
The issue is that `Ogma` should be installed from the private npm, and we cannot include a public link to it in this PR. We can provide you access to try the installation.

Alternatively, I can replace the version in `package.json` with the private npm link template, where it would be more obvious that you would need to obtain the access to it from Linkurious